driver/node.go

```diff
@@ -95,7 +90,7 @@ func (d *Driver) NodeStageVolume(ctx context.Context, req *csi.NodeStageVolumeRe
 		"method":              "node_stage_volume",
 	})
 
-	_, ok := req.VolumeContext[annNoFormatVolume]
+	_, ok := req.VolumeContext[d.annNoFormatVolume]
```

Comment:
I wonder if should allow both old and new annotations here.  That might make user-facing documentation easier and reduce the pain caused by confusing the two.

Reply:
Good idea. I've implemented this in the latest push.

I hard-coded the new and legacy driver names for this rather than using the flag - seems sensible given that we're supporting both, but let me know if you have a nicer way to do it.